deployment/nixops/secrets.json.example

```diff
@@ -11,5 +11,7 @@
         },
         "pagerdutyKey": "somekey",
         "googleClientID": "somekey",
-        "googleClientSecret": "somekey"
-}
\ No newline at end of file
+        "googleClientSecret": "somekey",
+        "deploymentRef": "somekey",
+        "githubToken": "somekey"
+}
```

Comment:
The reason I didn't add these is that they are optional, there are a few keys for the deployment server and if they are not there the deployment server is not deployed.

Reply:
Sadly, I learned last week that they're not optional. Deployment fails without them... :-(

Reply:
ouch! ok well this PR is fine, I'll also look in to better ways to do this at some point